.github/workflows/stale-prs.yml

```
@@ -0,0 +1,97 @@
+        id: compile_report
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const fiveDaysAgo = new Date(new Date().setDate(new Date().getDate() - 5)); 
+            const threeDaysAgo = new Date(new Date().setDate(new Date().getDate() - 3));
+            let report = '';
+            let noReviewsReport = '';
+            let noChangesReport = '';
+            let needsReviews = '';
+            let needsChanges = '';
+            let needsMerge = '';
+
+            const prs = await github.paginate(github.rest.pulls.list, {
+              owner: 'rancher',
+              repo: 'shepherd',
+              state: 'open',
+              per_page: 100,
+            });
+
+            for (const pr of prs) {
+              const createdDate = new Date(pr.created_at); 
+              const updatedDate = new Date(pr.updated_at); 
+              const reviews = await github.rest.pulls.listReviews({
+                owner: 'rancher',
+                repo: 'shepherd',
+                pull_number: pr.number,
+                per_page: 100,
+              });
+              let lastReviewDate = null
+              let lastReviewStateOne = ""
+              let lastReviewStateTwo = ""
+              let numberOfApprovedState = 0
+              statesCompare = ["COMMENTED", "CHANGES_REQUESTED"]
+              
+              if (reviews.data.length == 0 && createdDate < fiveDaysAgo) {
+                noReviewsReport += `${pr.title} #${pr.number} ${pr.html_url} ${pr.user.login}\n`;
+              } 
+              else {
+                  for (const review of reviews.data) {
+                    const reviewDate = new Date(review.submitted_at);
+                    if(!lastReviewDate || reviewDate > lastReviewDate) {
+                      lastReviewDate = reviewDate
+                      lastReviewStateTwo = lastReviewStateOne
+                      lastReviewStateOne = review.state
+                      if ( review.state === "APPROVED"){
+                        numberOfApprovedState+=1
+                      }
+                  }
+                }
+              }
+              if (lastReviewDate && (lastReviewDate < threeDaysAgo && updatedDate <= lastReviewDate) && statesCompare.includes(lastReviewStateOne) && statesCompare.includes(lastReviewStateTwo)) {
+                needsChanges += `${pr.title} (#${pr.number}) ${pr.html_url} ${pr.user.login} \n`;
+                
+              } else if (lastReviewDate && (updatedDate > lastReviewDate) && (updatedDate < threeDaysAgo)){
+                needsReviews += `${pr.title} (#${pr.number}) ${pr.html_url} ${pr.user.login} \n`;
+
+              }  else if (lastReviewDate && numberOfApprovedState >=2 && (lastReviewDate < threeDaysAgo) && !statesCompare.includes(lastReviewStateOne)) {
+                needsMerge += `${pr.title} (#${pr.number}) ${pr.html_url} ${pr.user.login} \n`;
+              }
+            }
+            if ( noReviewsReport) {
```

Just a nit here about spacing it
```suggestion
            if ( noReviewsReport ) {
```